Extensions/BitmapText/bitmaptextruntimeobject-pixi-renderer.js

```diff
@@ -73,37 +84,56 @@ gdjs.BitmapTextRuntimeObjectPixiRenderer.prototype.getRendererObject = function(
   return this._pixiObject;
 };
 
+// REVIEW comparé le wrap avec celui de l'IDE
 gdjs.BitmapTextRuntimeObjectPixiRenderer.prototype.updateWordWrap = function() {
   this._pixiObject.style.wordWrap = this._object._wordWrap;
   this._pixiObject.dirty = true;
   this.updatePosition();
 };
 
+// REVIEW comparé le wrap avec celui de l'IDE
 gdjs.BitmapTextRuntimeObjectPixiRenderer.prototype.updateWrappingWidth = function() {
-  this._pixiObject.maxWidth = this._object._wrappingWidth;
+  if (this._object.wordWrap) {
+    this._pixiObject.maxWidth = width;
+  } else {
+    this._pixiObject.maxWidth = 0;
+  }
+
+    
+
+
   this._pixiObject.dirty = true;
+
+ 
+  this._pixiObject.maxWidth = this._object._wrappingWidth;
   this.updatePosition();
+ 
 };
 
+// REVIEW  text - seem OK
 gdjs.BitmapTextRuntimeObjectPixiRenderer.prototype.updateTextContent = function() {
   this._pixiObject.text = this._object._text;
   this.updatePosition();
 };
 
+// REVIEW fill - seem OK
 gdjs.BitmapTextRuntimeObjectPixiRenderer.prototype.updateColor = function() {
   this.style.fill = this._object._color;
   this._pixiObject.dirty = true;
 };
 
+// REVIEW align - seem OK
 gdjs.BitmapTextRuntimeObjectPixiRenderer.prototype.updateAlignment = function() {
   this._pixiObject.align = this._object._align;
-  this._pixiObject.dirty = true;
 };
+
+// REVIEW fontFamily - seem OK
 gdjs.BitmapTextRuntimeObjectPixiRenderer.prototype.updateFontFamily = function() {
   this.style.fontFamily = this._object._runtimeScene
     .getGame()
     .getFontManager()
     .getFontFamily(this._object._fontFamily);
+  //update bitmapFont
```

Comment:
Yep, surely you need to do something similar to what is done in the constructor (reinstantiate a new font).